aldryn_people/search_indexes.py

```diff
@@ -15,7 +16,8 @@ class PeopleIndex(get_index_base()):
     INDEX_TITLE = True
 
     def get_title(self, obj):
-        return obj.name
+        with switch_language(obj):
+            return obj.name
```

Comment:
@mkoistinen asked in other PR:

> Does this do anything? It would seem to me that if the obj is in language XX already, then returning its name would already be in lang XX, right?

The `override`  used in `SearchIndex.prepare` does not change object language every time but is safe to use because it's Django thing. However it set the thread language. In some cases just get the `obj.name` was not giving me the correct translation and so the use of `switch_language` assert that attribute will be in thread language.

 Add the `switch_language` for parler based projects assert that object